NoMAD-ADAuth/KerbUtil.h

```diff
@@ -23,6 +23,7 @@ extern OSStatus SecKeychainItemSetAccessWithPassword(SecKeychainItemRef item, Se
 
 - (void)getKerberosCredentials:(NSString *)password :(NSString *)userPrincipal completion:(void(^)(NSString *))callback;
 - (NSString *)getKerbCredentials:(NSString *)password :(NSString *)userPrincipal;
+- (void)changeKerberosPassword:(NSString *)oldPassword :(NSString *)newPassword :(NSString *)userPrincipal completion:(void(^)(NSString *))callback;
 - (NSString *)changeKerbPassword:(NSString *)oldPassword :(NSString *)newPassword :(NSString *)userPrincipal;
```

Comment:
@macshome I left this previous delegate-based version of the changePassword function as-is, just like with authenticate in my previous PR, so that the updates in this PR are additive. Since we won't be using the old version of changePassword though, I could mark it as deprecated if you'd prefer; if so, please let me know

Reply:
I was just curious. Not breaking current API is good!